code/controllers/subsystem/dbcore.dm

```diff
@@ -1,9 +1,10 @@
+#define FAILED_DB_CONNECTION_CUTOFF 5

```

Comment:
only used in a single place and the proc comments on the exact time, might as well just put the number in the proc instead.